mkosi

```diff
@@ -2889,6 +2889,7 @@ def install_unified_kernel(args: CommandLineArguments,
                       "-v",
                       "--no-hostonly",
                       "--uefi",
+                      "--uefi-stub", "/usr/lib/systemd/boot/efi/linuxx64.efi.stub",
```

Comment:
The PR is already quite minimal at this point, but might it be whittled down a bit further? 

I just looked at the the dracut man page and the default search path for `uefi_stub` is `/lib/systemd`. Since on a Debian with merged usr `/lib` is symlink to `/usr/lib` this line shouldn't be strictly necessary, right?